programs-remote-listings/templates/archive-program.php

```diff
@@ -46,7 +46,7 @@
         <?php endif; ?>
 
         <?php if ( $program->text) : ?>
-            <div class="rs-program-excerpt"><?php echo wp_trim_words($program->text, 100); ?></div>
+            <div class="rs-program-excerpt"><?php echo $RS_Connect->excerpt($program->text); ?></div>
```

Comment:
just my opinion: I would have made this static, RS_Connect::excerpt() and I would have passed in the default 100 instead of hard coding it in the class. but it's fine.